<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>736</width>
     <height>418</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="bg_frame">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.341, x2:1, y2:0.897, stop:0 rgba(97, 152, 255, 255), stop:0.514124 rgba(186, 38, 175, 255), stop:1 rgba(255, 0, 0, 255));
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="bg_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="title_bar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="title">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="title_label">
               <property name="font">
                <font>
                 <family>Montserrat Semi Bold</family>
                 <pointsize>22</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PYLEARN</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="btn_frame">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: none;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_maximize">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Maximize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(85, 255, 127);
}
QPushButton:hover {
	background-color: rgba(85, 255, 127, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_minimize">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Minimize</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {
	background-color: rgba(255, 170, 0, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_quit">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>17</width>
                 <height>17</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Close</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {
	background-color: rgba(255, 0, 0, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="content_frame">
         <property name="font">
          <font>
           <family>iCiel Effra</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="name_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="name_label">
               <property name="font">
                <font>
                 <family>iCiel Effra</family>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tên bài tập:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="name_entry">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>80</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Corbel</family>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="num_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QLabel" name="num_label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>iCiel Effra</family>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Số bài tập:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="num_entry">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>80</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Corbel</family>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QFrame" name="confirm_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item alignment="Qt::AlignRight">
              <widget class="QPushButton" name="confirm_button">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color: rgb(102, 146, 251);
}

QPushButton:hover {
	background-color: rgba(102, 146, 251, 200);
}</string>
               </property>
               <property name="text">
                <string>OK</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="frame_grip">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
